package.json

```diff
@@ -82,7 +83,7 @@
     "react-native-pages": "^0.9.0",
     "react-native-picker-select": "^6.3.1",
     "react-native-pure-chart": "0.0.24",
-    "react-native-reanimated": "~2.2.0",
+    "react-native-reanimated": "1",
```

Comment:
I understand this change is to rid off a warning. Do you know if downgrading from 2 to 1 has any side effect? Is version 2 better in some way?

Reply:
Effect mainly in drag list, other animated views are import from native. I will check one more time.